.github/workflows/shellcheck-image.yaml

```diff
@@ -39,5 +39,14 @@ jobs:
           registry: quay.io/redhat-pipeline-service
           username: ${{ secrets.QUAY_USERNAME }}
           password: ${{ secrets.QUAY_TOKEN }}
+      - name: Push to ghcr.io
+        id: push-to-ghcr
+        uses: redhat-actions/push-to-registry@v2
+        with:
+          image: ${{ steps.build-image.outputs.image }}
+          tags: ${{ steps.build-image.outputs.tags }} ${{ github.ref_name }}
+          registry: ghcr.io/${{ github.repository_owner }}
```

Comment:
shellcheck image is for testing, shall we expect those images to be under something like test/images?

Reply:
ghcr is an image registry so `images` seems a bit redundant, but I've added `test` to the path.